src/map/status_effect_container.cpp

```diff
@@ -1448,6 +1450,100 @@ void CStatusEffectContainer::CheckEffectsExpiry(time_point tick)
     DeleteStatusEffects();
 }
 
+
+void CStatusEffectContainer::TickAuras(time_point tick)
```

Comment:
This is the main chunk right here, anyone else reviewing: This is the bit that needs the most scrutiny, as this is the part that will affect performance. (Along with getAllInRange, but not that one so much)

Reply:
It works, im not gonna doubt that, and its stable, tested lots and does the job